src/main/java/ch/njol/skript/Skript.java

```diff
@@ -681,22 +677,71 @@ protected void afterErrors() {
 								TestTracker.testStarted("run scripts");
 								TestTracker.testFailed("exception was thrown during execution");
 							}
+							if (TestMode.JUNIT) {
+								SkriptLogger.setVerbosity(Verbosity.DEBUG);
+								info("Running all JUnit tests...");
+								long milliseconds = 0, tests = 0, fails = 0, ignored = 0, size = 0;
```

Comment:
```suggestion
								long milliseconds = 0;
								long tests = 0;
								long fails = 0;
								long ignored = 0;
								long size = 0;
```

Reply:
I don't agree with this change.

Reply:
That's ok, although field inlining like this is generally very bad, it isn't strictly against Skript's code conventions.